client/src/components/Applications/AppForm/AppForm.jsx

```diff
@@ -682,52 +736,46 @@ export const AppSchemaForm = ({ app }) => {
                   <div className="appSchema-header">
                     <span>Configuration</span>
                   </div>
+                  {app.execSystems &&
+                    Object.keys(execSystemsWithAllocation).length > 1 && (
+                      <FormField
+                        label="Execution System"
+                        name="execSystemId"
+                        description="Select the system this job will execute on."
+                        type="select"
+                        required
+                      >
+                        {execSystemsWithAllocation
+                          .sort()
+                          .map((exec_system_id) => (
+                            <option key={exec_system_id} value={exec_system_id}>
+                              {exec_system_id}
```

Comment:
Nice. Should we use a semicolon instead of a hyphen to separate the name from id? Might reduce some confusion since colons are not allowed in system ids. i.e. `Frontera: frontera1-cyemparala` instead of `Frontera - frontera1-cyemparala`

Reply:
Yes, did that I now switched to `:`.
Also, while doing this found that I mistakenly skipped jest unit tests, fixed that. And added coverage

Reply:
There's also a third axis of info that may be useful: the system's host. Should we present this to #cep-design for feedback?

Reply:
Yes, will sync up with cep-design